chalicelib/lib/slack.py

```diff
@@ -6,16 +6,31 @@
 import hmac
 import hashlib
 import base64
-import slack
 
 log = logging.getLogger(__name__)
 
 
 class Slack:
 
+    slack_api = "https://slack.com/api"
+
     def __init__(self, slack_token):
         self.slack_token = slack_token
-        self.client = slack.WebClient(token=slack_token)
+        self.headers = {'Content-Type': 'application/json; charset=utf-8', 'Authorization': f'Bearer {self.slack_token}'}
```

Comment:
Really long line and using single quotes

Reply:
do we even need to do self.slack = slack_token ?
if we are creating the header only and not using the token in other parts?
in example:
```
def __init__(self, slack_token):
        self.headers = {'Content-Type': 'application/json; charset=utf-8', 'Authorization': f'Bearer {slack_token}'}
```

Reply:
@kamigerami I I wanted to have it left as an attribute since it might be useful later. But I will change that since it's probably better to do that when the need actually arises.

Reply:
@tommymalmqvist 122 chars isn't that long is it? But sure I can make it shorter